src/Plugin/resource/Field/ResourceFieldEntity.php

```diff
@@ -280,7 +280,7 @@ public function access($op, DataInterpreterInterface $interpreter) {
       return FALSE;
     }
 
-    $access = $property_wrapper->access($op, $account);
+    $access = $interpreter->getWrapper()->value() !== FALSE && $property_wrapper->access($op, $account);
```

Comment:
If `$interpreter->getWrapper()->value() === FALSE` it means that the entity could not be loaded, thus checking properties on it will result in errors.

Ex: this happens when the embedded author is the anonymous user. Doing `user_load(0)` returns `FALSE`.